dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -170,21 +171,22 @@ export default defineComponent({
      * @param event event
      */
     editDataset(event: Event) {
-      if (this.singleDataMetaInfoToDisplay) {
+      if (this.mapOfReportingPeriodToActiveDataset.size > 1) {
+        const panel = this.$refs.reportingPeriodsOverlayPanel as OverlayPanel;
+        if (panel) {
+          panel.toggle(event);
+        }
+      } else if (this.singleDataMetaInfoToDisplay) {
         this.gotoUpdateForm(
           this.singleDataMetaInfoToDisplay.companyId,
           this.singleDataMetaInfoToDisplay.dataType,
           this.singleDataMetaInfoToDisplay.dataId
         );
```

Comment:
The order of those two code blocks has to be reverted to the way it was in `main`.
We don't want that the panel opens, if `this.singleDataMetaInfoToDisplay` exists.
Reason: If  `this.singleDataMetaInfoToDisplay` exists, that you are ona single-view-framework-page (e.g. Eu-Taxo-FInancials) and there when you click on "EDIT", you want to be redirected to the Edit-page for the currently selected reporting period.

Reply:
Done.